### DISCUSSION TITLE
Issues when creating a second table

### DISCUSSION THREAD

**Discussion by knoxjeffrey** [supabase/supabase#2914] [Questions]
Hello, I've been using Supabase for a dashboard on my personal website and I really love the product. I'm wanting to create an additional table for a new dashboard but I seem to be getting errors.

When I create the table through the dashboard UI, I get the messages,

- Creating table ....
- Adding primary key ...
- Table is good to go ...

but as soon as I get that success message, I get another message saying that the table has been deleted. However, it's not deleted and I can still see it in my list of tables. If I add a new column, it saves the details but then once again tells me that the table has been deleted.

I then tried to dump the database to my local computer and I get an error:

```
pg_dump: error: query was: LOCK TABLE public.my_table_name IN ACCESS SHARE MODE
```
I'm not sure what is happening because I'm creating the table in the same way as I did for the first one. Just wondering if anyone else has seen this?

Thanks,
Jeff

**Comment by silentworks** [comment]
I haven't seen this issue before. It's probably best to contact the Supabase team on this one by using the Feedback widget inside the dashboard or email beta@supabase.io.

**Reply by knoxjeffrey** [reply]
Cheers @silentworks, I've just logged that in the widget for reporting a bug.

**Reply by knoxjeffrey** [reply]
I'll feedback on here once I hear back with an answer
